app/views/layouts/main.hbs

```diff
@@ -34,23 +34,28 @@
             .config(function($interpolateProvider) {
                 $interpolateProvider.startSymbol('{[{').endSymbol('}]}');
             })
+            .config(function($httpProvider) {
+                // disable browser caching of GET requests
+                $httpProvider.defaults.headers.get = $httpProvider.defaults.headers.get || {};
+                $httpProvider.defaults.headers.get['Cache-Control'] = 'no-cache';
+                $httpProvider.defaults.headers.get['Pragma'] = 'no-cache';
+            })
             .constant('constants', {
                 API_ROOT: '/api'
-            });
+            })
+            .value('currentUserId', '{{user.id}}');
```

Comment:
This is where I'm dumping the userId that was fetched by the server to the client.